File: Java/DailyFlash/August/18August/Program3.java


class CheckChar  {

	public static void main(String[] args) {

		char ch = '*';

		if(ch>=48 && ch<=57)
			System.out.println(ch+" is a Digit");
		else if((ch>=65 && ch<=90) || (ch>=97 && ch<=122))
			System.out.println(ch+" is a Alphabet");
		else
			System.out.println(ch+" is a Special Character");
	}
}

File: PPA/DailyFlash/August/18August/Program3.c


#include<stdio.h>

void main() {

	char ch = '*';

	if(ch>=65 && ch<=90 || ch>=97 && ch<=122)
		printf("%c is an alphabet\n",ch);
	else if(ch>=48 && ch<=57)
		printf("%c is an Digit\n",ch);
	else
		printf("%c is a Special Character\n",ch);
}
